client/Modals/DisplayFocusedImage.jsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable react/prop-types */
+import React, { Component } from 'react';
+
+class DisplayFocusedImage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      dynamicPhotoIndex: 0,
+    };
+  }
+
+  componentDidMount() {
+    const { photoIndex } = this.props;
+    this.setState({
+      dynamicPhotoIndex: photoIndex,
+    });
+  }
```

Comment:
Does this need to be in componentDidMount? Maybe you have it there for a reason I don't see, but seems like you could just put the same code in the constructor.

Reply:
You are right... Ill change it. Thank You!